<?xml version="1.0"?>
<launch>

  <node pkg="lidart_gap_finding" name="find_gap_node" type="find_gap_smooth.py" output="screen"/>
  <node pkg="lidart_wall_following" name="find_intersection_node" type="find_intersection_node" output="screen"/>

  <node pkg="rviz" name="rviz" type="rviz" output="screen"/>
  <!-- <node pkg="rqt_console" name="rqt_console" type="rqt_console" output="screen"/> -->

</launch>
